### DISCUSSION TITLE
is it possible to assume a default data format?

### DISCUSSION THREAD

**Discussion by zhiqiangZHAO** [rometools/rome#640] [Q&A]
Hi,

We are using Rome library to parse files from internet. Some file with only 
<feed xmlns:g=“http://base.google.com/ns/1.0")>

They don't give us the NS for Atom.
We are pretty sure they are all use "http://www.w3.org/2005/Atom".

Now Rome will return error b/c no parse found.
Is it possible to let Rome choose a default one for feed?

**Comment by PatrickGotthard** [comment]
Hi @zhiqiangZHAO,

currently this is not possible. Either you add the Namespace to the feed before parsing or try to use the Atom parser directly. Here's a detailed explanation how ROME parses feeds: https://rome.readthedocs.io/en/latest/rome-core/technical-docs/feed-parsing/

Regards,
Patrick

**Comment by zhiqiangZHAO** [comment]
Thank you @PatrickGotthard 
It seems I can 
- implement a parser extend Atom10Parser
- override the isMyType()
